.circleci/ci-packages.txt

```diff
@@ -257,12 +257,8 @@ perl-YAML-1.24
 perl-YAML-LibYAML-0.83
 perl-YAML-PP-0.027
 psmisc-23.0
-python3-appdirs-1.4.3
-python3-packaging-20.3
 python3-pathspec-0.7.0
-python3-pyparsing-2.2.0
-python3-PyYAML-5.3.1
-python3-setuptools-40.5.0
+python3-PyYAML-5.4.1
 python3-yamllint-1.22.1
```

Comment:
not sure about those removals.

Reply:
there is also https://github.com/os-autoinst/openQA/pull/4267 . Let's see what happens.

Reply:
looks like changes in req for PyYAML.